### PR TITLE
Review pass on the spec and extracted the yaml file

### PR DESCRIPTION
**Pull request by jamsden**

### CODE REVIEW

File: specs/ldm/ldm-service.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/GetIncomingLinks'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/rdf+xml:
```
Comment (berezovskyi) [inline specs/ldm/ldm-service.yaml:31]: application/n-triples is a standard RDF representation too

File: specs/ldm/ldm-service.yaml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            text/turtle:
+            application/json+ld:
+        '400':
+            description:  Bad request
+
+  /rootservices:
```
Comment (berezovskyi) [inline specs/ldm/ldm-service.yaml:37]: OSLC has a standardized URL for discovery of the root service document.

File: specs/ldm/link-discovery-management-spec.html [hunk 3]
```diff
@@ -303,32 +316,39 @@ <h3>Terminology</h3>
         <dl class="spaced">
           <dt><dfn>Link</dfn></dt>
           <dd>
-            An assertion or RDF triple consisting of a (subject, predicate, object) that manifests an instance of a relationship between the referenced subject and object.
+            An assertion or RDF triple consisting of a (subject, predicate, object) that manifests an instance of a
```
Comment (berezovskyi) [inline specs/ldm/link-discovery-management-spec.html:319]: OSLC has Link class, which is also a valid representation of a link as a whole resource.

File: specs/ldm/link-discovery-management-spec.html [hunk 4]
```diff
@@ -303,32 +316,39 @@ <h3>Terminology</h3>
...
-            consumers and providers of OSLC resources in a distributed environment based on REST. A particular
-            application component could be a client for some OSLC domain services and a server for the same or another
-            domain.
+            A server implementing the OSLC Link Discovery Management specification. OSLC LDM clients consume services
+            provided by LDM Servers to discover incoming links. The use of the terms Client and Server are intended to
+            distinguish typical consumers and providers of OSLC resources in a distributed environment based on REST. A
+            particular application component could be a client for some OSLC domain services and a server for the same
+            or another domain.
           </dd>
           <dt>Concept Resource</dt>
```
Comment (berezovskyi) [inline specs/ldm/link-discovery-management-spec.html:337]: Why is this spec redefining a definition from another spec?

File: specs/ldm/ldm-service.yaml (hunk 1 above)
Comment (MartinUlrich) [reply specs/ldm/ldm-service.yaml:31]: added to yaml with next pr

File: specs/ldm/ldm-service.yaml (hunk 2 above)
Comment (MartinUlrich) [reply specs/ldm/ldm-service.yaml:37]: According to LDM Spec: LDM Service URI is configured explicitly in each LDM client. Therefore no dedicated discovery protocol is needed.

File: specs/ldm/link-discovery-management-spec.html (hunk 3 above)
Comment (MartinUlrich) [reply specs/ldm/link-discovery-management-spec.html:319]: not aware of that and couldn't find any reference. Could you please provide the source for "oslc:link"?

File: specs/ldm/link-discovery-management-spec.html (hunk 4 above)
Comment (MartinUlrich) [reply specs/ldm/link-discovery-management-spec.html:337]: done - referred to OSLC-Server

**Pull request merged**